outline/data_structures.md

```diff
@@ -53,7 +53,7 @@ you have programmed in another language, these might have been
 called arrays in that language.
 {: ng-show="block21" .description}
 
-#### Compartments like structure <button class="link" ng-model="block22" ng-click="block22=!block22">Details</button>
+#### Compartment-like structure <button class="link" ng-model="block22" ng-click="block22=!block22">Details</button>
```

Comment:
@davidchambers yes it is! ClojureBridge's slides are directly rendered from markdown files. To make it like slides, current curriculum markdowns have such sort of notations here and there. I mentioned this at ClojureBridge Workshop ml, https://groups.google.com/forum/#!topic/clojurebridge-workshops/fdh3-r5anJw